extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -71,6 +73,9 @@ export class ConsentStateManager {
 
     /** @private {!Promise} */
     this.hasAllPurposeConsentsPromise_ = allPurposeConsentsDeferred.promise;
+
+    /** @public {?string} */
+    this.consentPageViewID64 = once(() => getRandomString64(this.ampdoc_.win));
```

Comment:
1. Don't need `once`.
2. consentPageViewId64 (lowercase `d`)

Reply:
@dmanek We absolutely need `once()`. This has two purposes:

1. We don't create the value immediately, but lazily, hence a function.
2. We'd like to preserve the value, so we `once()` it so that it's not calculated twice.

Reply:
Doesn't this get created once when the constructor is called either way?

Reply:
We create the function, but we don't calculate until the function it's called.

Reply:
Ah I missed that this was still a ref to a function to be called later. Thanks for clarifying! I think the type is wrong then?

Reply:
Yeah, made a comment. thanks